### ISSUE TITLE
[BUG] [Linter] Path-related error in version v0.6.5

### ISSUE THREAD

**Issue by yaegassy** [Riverside-Healthcare/djLint#151]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info

 - OS: Mac OS X 10.15.7
 - Python Version: Python 3.10.0
 - djLint Version: 0.6.5

## Issue

This problem may be a very rare case, but it occurred in my environment, so I will report the issue. (Perhaps there is a simpler case for reproduction.)

### Create a dedicated venv for djlint and install djlint

```
mkdir -p /tmp/djlint-bin
python3 -m venv /tmp/djlint-bin/venv
/tmp/djlint-bin/venv/bin/pip install djlint
```

### Clone the djLint repository

```
cd /tmp
git clone git@github.com:Riverside-Healthcare/djLint.git
```

### Go to the djLint directory and run djlint

```
cd djLint
cat tests/bad.html | /tmp/djlint-bin/venv/bin/djlint -
# or
cat /tmp/djLint/tests/bad.html | /tmp/djlint-bin/venv/bin/djlint -
```

### Output(Error)

```
djLint$ cat /tmp/djLint/tests/bad.html | /tmp/djlint-bin/venv/bin/djlint -

Linting 1/1 files ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 00:00

Traceback (most recent call last):
  File "/tmp/djlint-bin/venv/bin/djlint", line 8, in <module>
    sys.exit(main())
  File "/private/tmp/djlint-bin/venv/lib/python3.10/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/private/tmp/djlint-bin/venv/lib/python3.10/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/private/tmp/djlint-bin/venv/lib/python3.10/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/private/tmp/djlint-bin/venv/lib/python3.10/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/private/tmp/djlint-bin/venv/lib/python3.10/site-packages/djlint/__init__.py", line 226, in main
    if bool(print_output(config, file_errors, len(file_list))):
  File "/private/tmp/djlint-bin/venv/lib/python3.10/site-packages/djlint/output.py", line 34, in print_output
    lint_error_count += build_output(error["lint_message"], config)
  File "/private/tmp/djlint-bin/venv/lib/python3.10/site-packages/djlint/output.py", line 88, in build_output
    f"{Fore.GREEN}{Style.BRIGHT}\n{build_relative_path(list(error.keys())[0],config.project_root)}\n{Style.DIM}"
  File "/private/tmp/djlint-bin/venv/lib/python3.10/site-packages/djlint/output.py", line 72, in build_relative_path
    return str(Path(url).relative_to(project_root.resolve()))
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/pathlib.py", line 816, in relative_to
    raise ValueError("{!r} is not in the subpath of {!r}"
ValueError: '/private/var/folders/v_/b5wzz2q96q5cqgdtl3fq066r0000gn/T/tmp9a84vnwh' is not in the subpath of '/private/tmp/djLint' OR one path is relative and the other is absolute.
```

**Comment by christopherpickering** [comment]
I added a fix, but not sure how to test it. Do you mind pulling master and seeing if your problem is gone?

**Comment by yaegassy** [comment]
I'll check back later about the fix!

---

First, I tried djlint v0.6.5 with the repository source of a product called wagtail, and was able to reproduce it.

I don't know how to test it, but it seems that the presence or absence of the `.git/` directory has some effect.

## Prepare

```
cd /tmp
git clone git@github.com:wagtail/wagtail.git --depth=1
cd wagtail

python3 -m venv venv
source venv/bin/activate

pip install djlint
cat wagtail/admin/templates/wagtailadmin/base.html | djlint -
```

## [NG] Error

```
(venv) wagtail$ cat wagtail/admin/templates/wagtailadmin/base.html | djlint -

Linting 1/1 files ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 00:00

Traceback (most recent call last):
  File "/private/tmp/wagtail/venv/bin/djlint", line 8, in <module>
    sys.exit(main())
  File "/private/tmp/wagtail/venv/lib/python3.10/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/private/tmp/wagtail/venv/lib/python3.10/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/private/tmp/wagtail/venv/lib/python3.10/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/private/tmp/wagtail/venv/lib/python3.10/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/private/tmp/wagtail/venv/lib/python3.10/site-packages/djlint/__init__.py", line 226, in main
    if bool(print_output(config, file_errors, len(file_list))):
  File "/private/tmp/wagtail/venv/lib/python3.10/site-packages/djlint/output.py", line 34, in print_output
    lint_error_count += build_output(error["lint_message"], config)
  File "/private/tmp/wagtail/venv/lib/python3.10/site-packages/djlint/output.py", line 88, in build_output
    f"{Fore.GREEN}{Style.BRIGHT}\n{build_relative_path(list(error.keys())[0],config.project_root)}\n{Style.DIM}"
  File "/private/tmp/wagtail/venv/lib/python3.10/site-packages/djlint/output.py", line 72, in build_relative_path
    return str(Path(url).relative_to(project_root.resolve()))
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/pathlib.py", line 816, in relative_to
    raise ValueError("{!r} is not in the subpath of {!r}"
ValueError: '/private/var/folders/v_/b5wzz2q96q5cqgdtl3fq066r0000gn/T/tmps9cuilgv' is not in the subpath of '/private/tmp/wagtail' OR one path is relative and the other is absolute.
```

## [OK] If the ".git" directory is not included

Change wagtail's `.git` directory to `dummy.git` and run. (`mv .git dummy.git`)

```
(venv) wagtail$ mv .git dummy.git
(venv) wagtail$ cat wagtail/admin/templates/wagtailadmin/base.html | djlint -

Linting 1/1 files ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 00:00


private/var/folders/v_/b5wzz2q96q5cqgdtl3fq066r0000gn/T/tmp438exd76
──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
H006 15:24 Img tag should have height and width attributes. <img class="wagtail-
T003 20:16 Endblock should have name. Ex: {% endblock body %}. {% endblock %}
T003 59:31 Endblock should have name. Ex: {% endblock body %}. {% endblock %}
T003 62:0 Endblock should have name. Ex: {% endblock body %}. {% endblock %}

Linted 1 file, found 4 errors.
```

**Comment by yaegassy** [comment]
I tried "master" in the directory structure of this comment. https://github.com/Riverside-Healthcare/djLint/issues/151#issuecomment-981451340

I tried it with the `.git` directory present.

No error was output, and it worked fine.

### Prepare

```bash
cd /tmp/wagtail
source venv/bin/activate

pip uninstall -y djlint
pip install -U git+https://github.com/Riverside-Healthcare/djLint

cat wagtail/admin/templates/wagtailadmin/base.html | djlint -
```

### [OK] Output

```
(venv) wagtail$ cat wagtail/admin/templates/wagtailadmin/base.html | djlint -

Linting 1/1 files ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 00:00


/private/var/folders/v_/b5wzz2q96q5cqgdtl3fq066r0000gn/T/tmpv9dk98jd
──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
H006 15:24 Img tag should have height and width attributes. <img class="wagtail-
T003 20:16 Endblock should have name. Ex: {% endblock body %}. {% endblock %}
T003 59:31 Endblock should have name. Ex: {% endblock body %}. {% endblock %}
T003 62:0 Endblock should have name. Ex: {% endblock body %}. {% endblock %}

Linted 1 file, found 4 errors.
```
